azext_edge/edge/common.py

```diff
@@ -37,10 +61,28 @@ class ResourceState(Enum):
     starting = "Starting"
     running = "Running"
     recovering = "Recovering"
+    succeeded = "Succeeded"
     failed = "Failed"
     ok = "OK"
     warn = "warn"
     error = "Error"
+    n_a = "N/A"
+
+    @classmethod
+    def map_to_color(cls, value) -> str:
+        return cls.map_to_status(value).color
+
+    @classmethod
+    def map_to_status(cls, value) -> CheckTaskStatus:
+        status_map = {
+            cls.starting.value: CheckTaskStatus.warning,
```

Comment:
what about running?

Reply:
everything else gets caught into the get default aka success